validator/client/runner.go

```diff
@@ -79,3 +85,13 @@ func run(ctx context.Context, v Validator) {
 		}
 	}
 }
+
+func handleAssignmentError(err error, slot uint64) {
+	if err == errutil.AssignmentNotFoundErr {
+		log.WithField(
+			"epoch", (slot*params.BeaconConfig().SlotsPerEpoch) - params.BeaconConfig().GenesisEpoch,
+		).Warn("Validator not yet assigned to epoch")
```

Comment:
Why can't we return an error that says validator not yet assigned from the RPC?

Reply:
Because there are two scenarios we want: either an error occurred when retrieving assignments in which we log an error, or assignments have not yet happened in the case we log a warning. You mean just logging the AssignmentNotFoundErr? Also, the error we receive from the RPC will not be capitalized, so I'd need to format it anyways with logs.